serverless-plugin/tests/alarms-dynamodb.test.js

```diff
@@ -0,0 +1,120 @@
+'use strict'
+
+const dynamoDbAlarms = require('../alarms-dynamodb')
+const CloudFormationTemplate = require('../cf-template')
+
+const { test } = require('tap')
+const { cloneDeep } = require('lodash')
+
+const defaultConfig = require('../default-config')
+const {
+  assertCommonAlarmProperties,
+  alarmNameToType,
+  createTestConfig,
+  defaultCfTemplate
+} = require('./testing-utils')
+
+const sls = {
+  cli: {
+    log: () => {}
+  }
+}
```

Comment:
I have moved this to the testing utils, so you could import it from there (i believe it's called `slsMock`).